src/stories/control/DarkModeButton.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { fireEvent, screen, render } from "@testing-library/react";
+import { DarkModeButton } from "./DarkModeButton";
+import { ColorModeProvider } from "@chakra-ui/react";
+
+describe("scenario", () => {
+
+  it("check if Light Mode", () => {
```

Comment:
Maybe rewrite this as "it should use light mode by default"? Consider `it` as part of your sentence, always, or alternatively, you could use `test`